testsuite/features/secondary/minkvm_guests.feature

```diff
@@ -415,3 +418,5 @@ Feature: Be able to manage KVM virtual machines via the GUI
     When I remove package "tftpboot-installation-SLE-15-SP4-x86_64" from this "server" without error control
     And I wait for "tftpboot-installation-SLE-15-SP4-x86_64" to be uninstalled on "server"
 
+  Scenario: Check for errors in Cobbler monitoring
+    When I check for Cobbler errors in the local logs
```

Comment:
This is a test, I would prefer:

  `Then the local logs of Cobbler should not contain errors